css_task2/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Home page design</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="container">
+
+        <nav class="navigation">
+            <a class="active" href="#">Компютърна графика с WebGL</a>
+            <a href="#">Програмиране с GO</a>
+            <a href="#">Програмиране с RUBY</a>
+            <a href="#">AGILE</a>
+        </nav>
+        <main>
+            <div class="buttons">
+                <button class="button-color button-shape button-margin" name="Login" value="Вход">Вход</button>
+                <button class="button-color button-shape" name="registration" value="Регистрация">Регистрация</button>
+            </div>
+            <div class="content">
+                <h1>Компютърна графика с WebGL</h1>
+                <hr>
+                <h3>доц. П. Бойчев</h3>
+                <p>След завършване на курса студентите ще могат да създават
+                    примерни статични изображения и тримерни анимации.
+                </p>
+                <p><span class="info">Тип: </span>задължителен избираем</p>
+                <p><span class="info">Курс: </span>1<sup>ви</sup></p>
+                <p><span class="info">Програма: </span>бакалавър</p>
+                <textarea placeholder="Добавяне на коментар"></textarea><br>
+
+                <button class="button-shape" name="cancel" value="Отказ">Отказ</button>
+                <button class="button-color button-shape" name="Public" value="Публикуване">Публикуване</button>
```

Comment:
You could wrap all the buttons in a div

Reply:
Maybe a better name for the button could be "publish"? Because public means something different.
```suggestion
                <button class="button-shape" name="cancel" value="Отказ">Отказ</button>
                <button class="button-color button-shape" name="publish" value="Публикуване">Публикуване</button>
```